libs/platform/src/lib/components/split-menu-button/split-menu-button.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, Input, Output, EventEmitter, ChangeDetectorRef } from '@angular/core';
+import { ButtonType } from '@fundamental-ngx/core';
+import { MenuComponent } from './../menu/menu.component';
+import { BaseComponent } from '../base';
+
+@Component({
+    selector: 'fdp-split-menu-button',
+    templateUrl: './split-menu-button.component.html',
+    styleUrls: ['split-menu-button.component.scss'],
+})
+export class SplitMenuButtonComponent extends BaseComponent {
+    /** Label for the first Button */
+    @Input()
+    buttonLabel: string;
+
+    /** reference to menu which will be controlled by split button */
+    @Input()
+    menu: MenuComponent;
+
+    /** The Sap-icon to include in the menu-button */
+    @Input()
+    icon: string;
+
+    /** The type of the button.
+     * 'Emphasized', 'Ghost', 'standard', 'positive', 'negative', 'transparent'
+     * Leave empty for default.'*/
+    @Input()
+    type: ButtonType;
+
+    /** Event sent when split-menu-button primary button is clicked */
+    @Output()
+    primarButtonClick: EventEmitter<MouseEvent | KeyboardEvent | TouchEvent> = new EventEmitter();
```

Comment:
Why only 3 events? Are we not targeting for mobile users? there are 6 more event types.